data/food/activities_to_create.json

```diff
@@ -176,6 +176,32 @@
       "Pig, conventional, national average, at farm gate {FR} U": "Pig, conventionnal, Bleu Blanc Coeur, at farm gate {FR} U"
     }
   },
+  {
+    "id": "pig-fr-organic",
+    "search": "Pig, conventional, national average, at farm gate",
+    "suffix": "[fr-organic], constructed by Ecobalyse",
+    "subactivities": [
+      "Pig, French average, finishing feed, conv prod, at farm gate {FR} U",
+      "Pig, French average, finishing feed, purchased part, conv prod, at farm gate {FR} U"
+    ],
+    "replace": {
+      "Soft wheat grain, basis scenario without lever, stored and transported, processing {FR} U": "wheat grain FR central region",
+      "Maize grain, conventional, 28% moisture, national average, animal feed, at farm gate {FR} U": "Grain maize, organic, animal feed, at farm gate {FR} U"
```

Comment:
Just so we're clear, this change only concerns finishing `Pig, French average, finishing feed, purchased part, conv prod, at farm gate {FR} U"` so about 17.6% of the 43.5% of total feed impact.
<img width="588" alt="Screenshot 2023-11-20 at 11 29 29" src="https://github.com/MTES-MCT/ecobalyse/assets/13175018/961aa406-889e-4ecf-b30d-14af63b3a4d1">

I also observe that `barley` has not replaced, I guess because there's no substitute ICV in the gsheet ? :
https://docs.google.com/spreadsheets/d/1NRmAMWcLqfmdTz6h1GoAVz4UFvEvwFBD/edit#gid=534292262